modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -510,17 +556,27 @@ describe('TSS Ecdsa Utils:', async function () {
       false,
     )];
 
+    const backupToUserPublicShare = Buffer.concat([
+      Buffer.from(backupKeyShare.nShares[1].y, 'hex'),
+      Buffer.from(backupKeyShare.nShares[1].chaincode, 'hex'),
+    ]).toString('hex');
+
+    const backupToBitgoPublicShare = Buffer.concat([
+      Buffer.from(backupKeyShare.nShares[3].y, 'hex'),
+      Buffer.from(backupKeyShare.nShares[3].chaincode, 'hex'),
+    ]).toString('hex');
```

Comment:
This was failing later during user key combine step, because the public share needs to be the sum of the y value + chaincode.